KnowWhereGraph/kwg-faceted-search
# Add autocompletion to the facets

This PR adds autocompletion to the input fields on the Hazard and Place tabs.

To Test
1. Check this branch out and deploy
2. For each facet, make sure that you see suggestions after entering two characters in each input
